scripts/format_java.sh

```diff
@@ -26,4 +26,4 @@ then
   chmod a+x .local/$JAR_NAME
 fi
 
-java -jar .local/$JAR_NAME -i **/*
```

Comment:
Was the original **/* not matching every file?

Reply:
Did it work for you? when I tried to run bash scripts/formate_java.sh, it would not check files recursively, it stopped at the second level of the directory. Should I use a different command?

Reply:
Here is the link where I find working after apply the change: https://github.com/google/google-java-format/issues/129

Reply:
I hadn't used this script before (assumed it was working), and looks like it's not working, so thanks for fixing it, sg.